README.md

```diff
@@ -160,33 +160,35 @@ If you'd like to clone the repository, you'll need to initialize and update the
 $ git clone --recursive git@github.com:atljeremy/JFCardSelectionViewController.git
 ```
 
-### Carthage
+### Installation
 
-`github "atljeremy/JFCardSelectionViewController" ~> 1.0`
-
-### Cocoapods
+#### CocoaPods
+You can use [CocoaPods](http://cocoapods.org/) to install `JFCardSelectionViewController` by adding it to your `Podfile`:
 
+```ruby
+platform :ios, '8.0'
+use_frameworks!
 `pod 'JFCardSelectionViewController', '~> 1.0.4'`
+```
+
+To get the full benefits import `JFCardSelectionViewController` wherever you import UIKit
+
+``` swift
+import UIKit
+import JFCardSelectionViewController
+```
+
+#### Carthage
+Create a `Cartfile` that lists the framework and run `carthage bootstrap`. Follow the [instructions](https://github.com/Carthage/Carthage#if-youre-building-for-ios) to add `$(SRCROOT)/Carthage/Build/iOS/YourLibrary.framework` to an iOS project.
```

Comment:
I believe this should actually be `carthage update`. `bootstrap` is meant for after you've ran `update` which fetches dependencies and updates the `Cartfile.resolved`. See #2 in the instructions you linked to which says:
```
2. Run carthage update. This will fetch dependencies into a Carthage/Checkouts folder, then build each one.
```

Also, can you update the line `$(SRCROOT)/Carthage/Build/iOS/YourLibrary.framework` to `$(SRCROOT)/Carthage/Build/iOS/JFCardSelectionViewController.framework`? This would be the accurate representation for someone adding this to the `carthage copy-frameworks` Run Script Build Phase.